woocommerce/woocommerce
# Fix: Show password toggle not working in checkout page

### All Submissions:






### Changes proposed in this Pull Request:


The toggle for the show password was bind to the `.password-input` class. The `.password-input` class was not added to the span wrapper of the password input in the check out page. So, the `.password-input` is added to the span wrapper by javascript, and the toggle code was also modified to handle that.

Closes #26271  .

### How to test the changes in this Pull Request:

1. Check out the master branch.
2. Make sure the password visible toggle is working on the registration page.
![Screenshot-20200425230103-1694x721](https://user-images.githubusercontent.com/8264719/80286025-b0b0e700-8748-11ea-8773-437d0e0ed801.png)
3. Make sure the password visible toggle doesn't work on the checkout page.
![Screenshot-20200425230213-873x410](https://user-images.githubusercontent.com/8264719/80286055-dfc75880-8748-11ea-9cd7-564c317e0dfd.png)
4. Check out the `fix/26271` branch.
5. Make sure the password visible toggle is working on the registration page, checkout page, and m account page as well.
![Screenshot-20200425231856-780x305](https://user-images.githubusercontent.com/8264719/80286440-2d44c500-874b-11ea-80bb-5ca7884b0592.png)
![Screenshot-20200425230448-951x604](https://user-images.githubusercontent.com/8264719/80286096-303eb600-8749-11ea-80c6-c938ba77e2be.png)

### Other information:




### Changelog entry

> Fix password visible toggle in the checkout page.
